### ISSUE TITLE
Hmean: 83.3%

### ISSUE THREAD

**Issue by Daniil-Osokin** [Wovchena/text-detection-fots.pytorch#11]
Hi!
You have finally managed to get 83% accuracy even without attention, great! My congratulations :tada:!

> BTW, [this](https://github.com/Wovchena/text-detection-fots.pytorch#weight-decay-history) should be 'Learning rate schedule' or something similar, weight decay is for [weights regularization](https://towardsdatascience.com/this-thing-called-weight-decay-a7cd4bcfccab).

**Comment by Wovchena** [comment]
Thank you :)
